extensions/android-paging3-coroutines/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.library'
+apply plugin: 'org.jetbrains.kotlin.android'
+
+archivesBaseName = 'sqldelight-android-paging3-coroutines'
```

Comment:
```suggestion
archivesBaseName = 'sqldelight-android-paging3'
```

Reply:
do you still envision us renaming this even though an RX module is coming, as well?

Reply:
your last comment on the rx gradle dependency makes me think you didn't realize that.

Reply:
Hmm. I think I would still rename since coroutines are part of the paging core whereas the Rx stuff is extra and opt-in.